content/en/bare-metal-machine.md

```diff
@@ -19,14 +19,14 @@ there is no artificial delay in translating operating system instructions to har
 
 ## How it helps
 
-By dedicating all compute resources of a computer to a single operating system, 
-you potentially provide the best possible performance to the operating system. 
+By dedicating all computer computing resources to a single operating system, 
+you can provide the best possible performance to the operating system. 
```

Comment:
I've never seen the `computer computing resources` expression. It looks odd and unfamiliar. I don't think this modification is an improvement.

Also, I think changing `potentially` to `can` does not improve the sentence. It is because we cannot be sure `you can provide the best possible performance`.

Reply:
Thanks for the suggestion and update. I agree with you if you think it should stay the same and you gave the reason. Please take care of the rest of the changes. Thanks again Seokho-son

Reply:
@pushpjeetcholkar
Yes please restore the sentence above.
Rest of the changes looks good to me :)

Reply:
@seokho-son, I have made the proposed change. Thanks for your guidance : )